tests/Feature/UserActivityTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\UserActivity;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class UserActivityTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testUserActivitiesAreCreated(): void
```

Comment:
Because there isn't a controller yet, I would just add your factory to the TestSeeder.php file, so you can use $this->seed(TestSeeder::class) in each test, which can auto create 10 and remove the need for this test.
 
The only thing I would maybe test, would be the `forUser()` method you create on the factory.

```php
// this will seed 10 
$this->seed(TestSeeder::class);
$user = User::factory()->createOne();
// this will add five more
UserActivity::factory(5)->forUser($user->id)->create();
// now you know there should be 10 created from the TestSeeder, and 5 for the User.
$all = UserActivity::all();
$five = [];
foreach($all as $one){
// if the user_id = $user->id, collect it into the $five array, and assert it contains five total.

Reply:
I will add testing for `forUser` but I plan to add the controller today.